### PR TITLE
feat(blog): adding blog post about k8s and smbs

### LINKED ISSUE #8: Footer and imprint

### PR DESCRIPTION
**Pull request by tgraupne**
# Description & Technical Solution

This PR adds a blog post about the specific challenges of SMBs migrating to Kubernetes.

# Checklist

- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] Already rebased against main branch.

# Screenshots

image

# Related issue

- Resolve #8

### CODE REVIEW

File: blogposts/k8s-smb.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+We advocated for migrating to a Kubernetes-based infrastructure as a foundational step to improve in multiple areas, addressing all the issues listed above.
+Key components of our solution include:
+
+- **Infrastructure as Code**: Using tools like Terraform or CDK, we transform infrastructure management into a more manageable, version-controlled process.
+- **State-of-the-Art Observability**: Integrating cutting-edge monitoring and logging tools that work seamlessly with Kubernetes, providing deeper insight into application performance and system health.
+- **CI/CD Pipeline Update**: Modernizing the continuous integration and deployment pipelines to support containerized applications, resulting in more flexible and efficient deployments.
+
+We recognize that all of the above can be implemented in more traditional VM-based infrastructures as well, however, our experience has shown that migrating to Kubernetes helps customers establishing an improved mindset for making important modernizations.
+Furthermore, in our experience, the above solutions reach their full potential when used in conjunction with Kubernetes.
```
Comment (kenodressel) [inline blogposts/k8s-smb.md:40]: Sollten wir hier nicht noch auf ein paar andere Kubernetes Vorteile eingehen?
- More efficient ressource usage === lower cost?
- Cloud Agnostic
- Better Secrets Management :wink:

**Review by kenodressel — COMMENTED** [review]
Liest sich sehr gut

File: blogposts/k8s-smb.md (hunk 1 above)
Comment (tgraupne) [reply blogposts/k8s-smb.md:40]: While you are 100% with those statements, my specific intention with this blog post is to shed light on the specific challenges SMBs face.

In our project experience at Unit 214, the valid arguments you've made are true, but in my opinion less heavy or important in the SMB case.

Nonetheless, it is still a good idea to at least reference them. 👍

Comment (tgraupne) [reply blogposts/k8s-smb.md:40]: Hey @kenodressel, was denkst du über das Update und meine Erläuterungen? 🙂

**Review by kenodressel — APPROVED** [review]

**Pull request merged**